### PR TITLE
Update Roary recipe

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR allows fixing a bug found in the Rscript used for generating the plots https://github.com/sanger-pathogens/Roary/pull/566
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/roary/build.sh [hunk 1]
```diff
@@ -20,4 +20,6 @@ else
     exit 1
 fi
 
+wget https://raw.githubusercontent.com/sanger-pathogens/Roary/master/bin/create_pan_genome_plots.R -P bin/
```
Comment (bgruening) [inline recipes/roary/build.sh:23]: $PREFIX/bin

File: recipes/roary/meta.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
+  number: 2
```
Comment (bgruening) [inline recipes/roary/meta.yaml:10]: And you need to add r-base and probably a few r-libraries to the run dependencies

Comment (gallardoalba) [reply recipes/roary/meta.yaml:10]: Ok, I'll check. Thanks!

**Comment by gallardoalba** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

File: recipes/roary/build.sh [hunk 3]
```diff
@@ -20,4 +20,6 @@ else
     exit 1
 fi
 
+wget https://raw.githubusercontent.com/sanger-pathogens/Roary/master/bin/create_pan_genome_plots.R -P $PREFIX/bin/
+
 chmod u+rwx $PREFIX/bin/*
```
Comment (dpryan79) [inline recipes/roary/build.sh:25]: I'm pretty sure this won't work. You can't change permissions on dependencies without throwing a file permissions error.

**Comment by dpryan79** [comment]
Ah, some graph dependency has changed its output.

**Comment by bgruening** [comment]
> Ah, some graph dependency has changed its output.

Oo, not sure if we will get upstream support here. Skipping the `make test`?

**Comment by bgruening** [comment]
@druvus do you maybe have time to look at this one? Thanks!

**Comment by druvus** [comment]
@bgruening I'll take a look

**Comment by druvus** [comment]
@BiocondaBot please update

**Comment by druvus** [comment]
[Bio::Perl not found](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=16639&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=2210)

**Pull request closed without merging**